library/src/main/java/com/novoda/downloadmanager/Batch.java

```diff
@@ -76,7 +76,7 @@ public Builder(DownloadBatchId downloadBatchId, String title) {
         }
 
         public Builder addFile(String fileUrl) {
-            String id = downloadBatchId.stringValue() + fileUrl;
+            String id = downloadBatchId.rawId() + fileUrl;
```

Comment:
can we change this variable to `rawId`? because you're creating an `id` from a variable called `id` right now. It would be clearer to create an `id` from a `rawId` imo

Reply:
8612e94